sdk/apimanagement/Azure.ResourceManager.ApiManagement/tests/Scenario/CertificateTests.cs

```diff
@@ -61,7 +61,11 @@ public async Task CRUD()
             X509Certificate2 cert = null;
             if (Mode != RecordedTestMode.Playback)
             {
+#if NET9_0_OR_GREATER
+                cert = X509CertificateLoader.LoadCertificate("./Resources/sdktest.cer");
```

Comment:
[SYSLIB0057: X509Certificate2 and X509Certificate constructors for binary and file content are obsolete](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0057)